### PR TITLE
LC-2918 user messages updated-4

### PR DESCRIPTION
**Pull request by neeraj-jain-cbo**

### CODE REVIEW

File: src/main/resources/templates/reactivate/accountReactivated.html [hunk 1]
```diff
@@ -4,13 +4,13 @@
 <div class="container">
     <div class="grid-row">
         <div class="govuk-box-highlight">
-            <h1 class="heading-xlarge">Account reactivation complete</h1>
-            <p class="font-medium">Your account has been successfully reactivated</p>
+            <h1 class="heading-xlarge">Account reactivation final step</h1>
```
Comment (jaylco) [inline src/main/resources/templates/reactivate/accountReactivated.html:7]: This doesn't read very well - maybe just trim it down to "Account reactivation"?

File: src/test/java/uk/gov/cabinetoffice/csl/controller/reactivation/ReactivationControllerTest.java [hunk 2]
```diff
@@ -61,27 +61,30 @@ public class ReactivationControllerTest {
...
         LocalDateTime requestedAt = LocalDateTime.now();
         reactivation.setRequestedAt(requestedAt);
-
+        String title = "Account reactivation pending";
+        String viewName = "reactivate/pendingReactivate";
         String reactivationEmailMessage = ("We&#39;ve sent you an email on %s with a link to reactivate your " +
                 "account.").formatted(utils.convertDateTimeFormat(requestedAt.toString()));
         LocalDateTime reactivationLinkExpiryDateTime = requestedAt.plusSeconds(reactivationValidityInSeconds);
-        String reactivationValidityMessage = "The link in the email will expire on %s."
+        String reactivationValidityMessage = ("The link in the email will expire on %s after which you will be " +
```
Comment (jaylco) [inline src/test/java/uk/gov/cabinetoffice/csl/controller/reactivation/ReactivationControllerTest.java:83]: Feels a little bit too wordy/complex

**Review by jaylco — COMMENTED** [review]
A few comments

File: src/main/resources/templates/reactivate/accountReactivated.html (hunk 1 above)
Comment (neeraj-jain-cbo) [reply src/main/resources/templates/reactivate/accountReactivated.html]: Done

File: src/test/java/uk/gov/cabinetoffice/csl/controller/reactivation/ReactivationControllerTest.java (hunk 2 above)
Comment (neeraj-jain-cbo) [reply src/test/java/uk/gov/cabinetoffice/csl/controller/reactivation/ReactivationControllerTest.java]: Done

**Review by jaylco — APPROVED** [review]
